code/modules/antagonists/antag_org/antag_org_syndicate.dm

```diff
@@ -81,15 +92,9 @@
 	name = "Steal Item (FAID)"
 	steal_list = list(/datum/theft_objective/blueprints, /datum/theft_objective/steal/documents)
 
-/// Hijack only orgs
-/datum/antag_org/syndicate/gorlex
+/datum/antag_org/syndicate/gorlex //Hijack only
 	name = "Gorlex Marauders"
-	intro_desc = "(TODO - LORE) You are a Gorlex operative. Get in, fuck shit up, get out with a fancy new shuttle. You know the drill."
-	forced_objective = /datum/objective/hijack
-	chaos_level = ORG_CHAOS_HIJACK
-
-/datum/antag_org/syndicate/waffle
-	name = "Waffle Company"
-	intro_desc = "(TODO - LORE) You are a Waffle Company operative. Get in, fuck shit up, get out with a fancy new shuttle. You know the drill."
+	intro_desc = "(TODO - LORE) You are an operative of the infamous Gorlex Marauders, a brutal and merciless gang of pirates and cutthroats. \
```

Comment:
Hi, you left a "(TODO - LORE)" in the final Gorlex Traitor Type

Reply:
This is fixed, nto sure why it wasnt there.